### PR TITLE
Merging various CLI fixes

### PR DESCRIPTION
**Pull request by avincigu**

### CODE REVIEW

File: cli/cli/src/group.py [hunk 1]
```diff
@@ -78,18 +81,19 @@ def _group_add_execute(self, args):
         # URL will be PUT http://hostaddress:hostport/groups/group_name?devices=devicelist
         url = client.get_base_url() + 'groups/' + args.group_name
         parameters = {'devices': (','.join(devices_to_add)), 'user': self.user}
-        #url = client.get_base_url() + 'groups/' + args.group_name + '?' + "devices=" + (','.join(devices_to_add)) + '&user=' + self.user
+        # url = client.get_base_url() + 'groups/' + args.group_name + '?' + "devices=" + (','.join(devices_to_add)) +
+        #  '&user=' + self.user
```
Comment (paul-amonson) [inline cli/cli/src/group.py:85]: Do we need to keep commented out code?

Comment (avincigu) [reply cli/cli/src/group.py:85]: I deeply dislike commented code, so if you are ok with getting rid of it I am too.

Comment (paul-amonson) [reply cli/cli/src/group.py]: Yes, remove it.

Comment (avincigu) [reply cli/cli/src/group.py]: Done

**Pull request merged**